src/assets/patterns/patterns.ts

```diff
@@ -20,7 +20,7 @@ export const Patterns = {
 
   paymantAmountPattern: '^[0-9]+$',
   sertificateMonthCount: '^[0-9]{1,2}$',
-  sertificateInitialValue: '^[0-9]{2,4}$',
+  sertificateInitialValue: /^\d*[.,]?\d{1,2}$/,
```

Comment:
With this pattern user can write more than max sum for certificate (Max sum is 9999.99)
![image](https://user-images.githubusercontent.com/104850911/219059417-53512069-d13e-431c-bc0c-3cf0c8cdffb0.png)
![image](https://user-images.githubusercontent.com/104850911/219059652-614861c7-8a9a-4708-b526-0a1c5ffd0941.png)

Reply:
Added changes, now it works correct.